### PR TITLE
Add bug report issue template

### PR DESCRIPTION
**Pull request by jradtilbrook**

### CODE REVIEW

**Review by james2791 — APPROVED** [review]
LGTM!

**Review by lizrabuya — APPROVED** [review]
Looks great!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (mcncl) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Will these be "click on" etc, given it's a bug with a Terraform provider?

Comment (jradtilbrook) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: @mcncl what about this?

```suggestion
Please provide a minimum `.tf` file to reproduce the issue.  
Provide any other instructions necessary, such as what fields/values to change to trigger the bug
```

Comment (mcncl) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: @jradtilbrook I think https://github.com/hashicorp/terraform-provider-aws/issues/31537 is a good reference for what to include.

That's the official AWS provider, so well used. Also a reasonable standard to hold ourselves to

**Review by mcncl — APPROVED** [review]

**Pull request merged**